ee-token

```diff
@@ -94,6 +108,68 @@ function is_valid_token() {
     [[ "${response_code}" == "302" ]]
 }
 
+function revoke() {
+    local token_or_email=$1
+    local key="downloadToken"
+    local question="Do you want to revoke the token '${token_or_email}'? (y/N): "
+
+    if [[ -z "${token_or_email}" ]]; then
+        if [[ -n "${GRAAL_EE_DOWNLOAD_TOKEN:-}" ]]; then
+            token_or_email="${GRAAL_EE_DOWNLOAD_TOKEN}"
+        else
+            # shellcheck source=/dev/null
+            source "${GU_CONFIG}"
+            if [[ -z "${GRAAL_EE_DOWNLOAD_TOKEN:-}" ]]; then
+                fail "Unable to detect \$GRAAL_EE_DOWNLOAD_TOKEN in '${GU_CONFIG}'. Please remove the file and try again."
```

Comment:
Is it clear which file the user should remove?

Reply:
`${GU_CONFIG}` is the file name, so I'd say yes :)